src/mn_setup.c

```diff
@@ -1286,6 +1286,11 @@ static void UpdateCenteredWeaponItem(void)
     DisableItem(!weapon_bobbing_pct, weap_settings1, "center_weapon");
 }
 
+static void UpdatePreBetaBFGItem(void)
```

Comment:
Hm, so now you cannot even occasionally select the Pre-Beta BFG if you're not on MBF complevel?

Reply:
Previously you could toggle it on/off in other complevels. But when you quit the game and restarted, it was always disabled as part of the initial demo header preparation (even if you aren't recording). So the menu would show "on" but it didn't actually work until toggled again. It seems like this pre-beta bfg setting shouldn't be changed at all once it's set in that initial demo preparation stage but I didn't restrict it that far.

Reply:
Reverted all my changes because it's out of scope.